ocaml/xapi/message_forwarding.ml

```diff
@@ -3110,6 +3110,11 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
       let local_fn = Local.PIF.set_property ~self ~name ~value in
       do_op_on ~local_fn ~__context ~host (fun session_id rpc -> Client.PIF.set_property rpc session_id self name value)
 
+    let set_disallow_unplug ~__context ~self ~value =
+      let pif_uuid = Db.PIF.get_uuid ~__context ~self in
+      info "PIF.set_disallow_unplug: PIF uuid = %s; value = %s" (pif_uuid) (string_of_bool value);
```

Comment:
You can use the existing `pif_uuid` helper: https://github.com/xapi-project/xen-api/pull/3485/files#diff-9a639b670fb7e2432894cc91df8b27fbR3101